src/renderers/shared/fiber/ReactFiberContext.js

```diff
@@ -57,6 +57,15 @@ function getUnmaskedContext(workInProgress : Fiber) : Object {
 }
 exports.getUnmaskedContext = getUnmaskedContext;
 
+function cacheContext(workInProgress : Fiber, unmaskedContext : Object, maskedContext : Object) {
+  const instance = workInProgress.stateNode;
+  if (instance) {
```

Comment:
PS. I'm happy to defer to your judgement here. You're more familiar with the project and the way things are done. Just sharing my thought process. 😄